core/src/test/kotlin/org/evomaster/core/output/dto/DtoWriterTest.kt

```diff
@@ -0,0 +1,176 @@
+package org.evomaster.core.output.dto
+
+import org.evomaster.core.EMConfig
+import org.evomaster.core.output.OutputFormat
+import org.evomaster.core.problem.rest.builder.RestActionBuilderV3
+import org.evomaster.core.problem.rest.schema.OpenApiAccess
+import org.evomaster.core.problem.rest.schema.RestSchema
+import org.evomaster.core.search.action.Action
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.hasItem
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.Paths
+
+class DtoWriterTest {
+
+    companion object {
+        val tmpTestSuitePath: Path = Paths.get(Files.createTempDirectory("dto-writer-test").toUri())
```

Comment:
rather then a tmp folder, should be somewhere under `target`, so that it is easy to double-check, and `maven clean` will get rid of it

Reply:
Hi @arcuri82! Hm, I was actually thinking these tests more for testing the data structures generated in memory and test the `DtoWriter` class, in order to assert that the actions and DTOs match a provided spec. 
Generated IO content is not that relevant here, instead for that I was thinking:
- E2E tests
- Unit tests focused on syntax testing the `JavaDtoWriter` and future `KotlinDtoWriter`

Reply:
but, using `tmpTestSuitePath`, aren't you writing to disk? any tmp file created during a test should go somewhere under `./target`.

Reply:
Done!